content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+contribute_url: 'https://github.com/mozilla/email-tabs'
+bug_report_url: 'https://github.com/mozilla/email-tabs/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/email-tabs'
+privacy_notice_url: 'https://github.com/mozilla/email-tabs/blob/master/docs/metrics.md'
+measurements:
+  - >
+    Email Tabs collects information about your use of the feature such as which email provider you choose, how often you interact with Email Tabs UI,  and whether you choose to send all tabs at once or just a selection.
+  - >
+    Email Tabs also collects information about the emails generated by the feature such as how many tabs you choose to send, the Email Tabs template you select, the count of recipients for Email Tabs emails, and whether the sender email address matches the recipient email address.
+  - >
+    Email Tabs does not collect any information about your email address, your email contacts or the URLS of any websites you visit or send while using Email Tabs.
+xpi_url: 'https://testpilot.firefox.com/files/email-tabs@mozilla.org/latest'
+addon_id: 'email-tabs@mozilla.org'
+gradient_start: '#008EA4'
+gradient_stop: '#31FFC3'
+details:
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-detail-1.jpg
+    copy: >
+      Create emails from your open tabs in just a few clicks.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-detail-2.jpg
+    copy: >
+      Email Tabs lets you send links, screenshots, and even full articles.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-detail-3.jpg
+    copy: >
+      Email Tabs automatically creates your email. Just add your recipients and send.
+tour_steps:
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-tour-0.jpg
+    copy: Access Email Tabs from your browser’s toolbar.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-tour-1.jpg
+    copy: You can choose to email all of your tabs, or just select a few.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-tour-2.jpg
+    copy: You can send links to your tabs, screenshots and links, or even full articles.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-tour-3.jpg
+    copy: Email Tabs will automatically format your email. Just add your recipients and send.
+  -
+    image: /static/images/experiments/email-tabs/email-tabs-tour-4.jpg
+    copy: You can always give us feedback or disable Email Tabs from Test Pilot.
+contributors:
+  -
+    display_name: 'Ian Bicking'
+    title: 'Software Engineer'
+    avatar: /static/images/experiments/page-shot/avatars/ian-bicking.jpg
+  -
+    display_name: 'Dave Justice'
+    title: 'Engineer'
+    avatar: /static/images/experiments/min-vid/avatars/dave-justice.jpg
+  -
+    display_name: 'Mark Liang'
+    title: 'Firefox UX'
+    avatar: /static/images/experiments/avatars/mliang.png
+created: '2017-11-12T14:00:00Z'
+launch_date: '2017-11-12T14:00:00Z'
```

Comment:
Should these be like, 2018?
`created` may not matter much (although I'll never be able to unsee), but the `launch_date` may give a weird side effect if we're hoping to deploy and have it hidden until Nov 12.

Reply:
@pdehaan yes. you are a hero.

Reply:
Ha, I missed that too. Great catch.